common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -137,7 +137,15 @@ class CapaFields(object):
             {"display_name": _("Finished"), "value": SHOWANSWER.FINISHED},
             {"display_name": _("Correct or Past Due"), "value": SHOWANSWER.CORRECT_OR_PAST_DUE},
             {"display_name": _("Past Due"), "value": SHOWANSWER.PAST_DUE},
-            {"display_name": _("Never"), "value": SHOWANSWER.NEVER}]
+            {"display_name": _("Never"), "value": SHOWANSWER.NEVER},
+            {"display_name": _("After # Attempts"), "value": SHOWANSWER.AFTER_ATTEMPTS}]
```

Comment:
I agree with Casey re: `#`; I think the objection here is the symbol chosen (not obvious/universal), not the idea.

I'd prefer something more explicit:

```python
{"display_name": _("After Some Number of Attempts"), "value": SHOWANSWER.AFTER_SOME_NUMBER_OF_ATTEMPTS}
```

"After Attempts" could imply "After _all_ Attempts" or "After _any_ Attempts".
We want it to be more clear that this is after an an _arbitrary_ number of attempts,
not after _all_ or _any_.